tests/test_telcameras_v2_ingress.py

```diff
@@ -24,6 +27,61 @@ def setUp(self):
         # Create an endpoint
         self.endpoint_obj = Endpoint.objects.create(url_key=self.endpoint_url_key)
 
+        # Create the sensor in the database
+        self.sensor = Sensors.objects.create(objectnummer='GAVM-01-Vondelstraat')
```

Comment:
Ik zie dit hier pas, en het is eigenlijk niet relevant voor deze specifieke PR, maar: een Django model heeft doorgaans een naam in enkelvoud. In dit geval zou dat `Sensor` moeten zijn, want één instance is één Sensor.